pandas/io/spss.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+from typing import Optional, Sequence, Union
+
+from pandas.core.api import DataFrame
+
+
+def read_spss(path: Union[str, Path],
+              usecols: Optional[Union[str, Sequence[str]]] = None,
+              categorical: bool = True) -> DataFrame:
+    """
+    Load an SPSS file from the file path, returning a DataFrame.
+
+    .. versionadded 0.25.0
+
+    Parameters
+    ----------
+    path : string or Path
+        File path
+    usecols : str or list-like or None
+        Return a subset of the columns. If None, return all columns.
+    categorical : bool
+        Convert categorical columns into pd.Categorical.
+
+    Returns
+    -------
+    DataFrame
+    """
+
+    from pyreadstat import read_sav
```

Comment:
Typically we do these in a `try / except ImportError` so we can raise with a nicer message that pyreadstat is an optional dependency.

Reply:
Done.